docs/index.html

```diff
@@ -0,0 +1,89 @@
+        <li>
+          <p>Suggested Videos</p>
+          <p>Trending Videos</p>
+        </li>
+        <li>
+          <p>Videos by Category</p>
+          <p>Comment Feed</p>
+        </li>
+      </ul>
+    </section>
+
+    <section class="forth">
+      <h1>Featured Technologies</h1>
+        <ul>
+          <li>
+            <p>Electron</p>
+            <p>Integration</p>
+          </li>
+          <li>
+            <p>React.js</p>
+            <p>Framework</p>
+          </li>
+          <li>
+            <p>Webpack</p>
+            <p>Codesplitting</p>
+          </li>
+        </ul>
+    </section>
+
+    <footer>
+      <a href="#">
+        <svg width="24px" height="24px" viewBox="0 0 24 24">
+          <g fill="#ffffff">
+            <path d="M18.9765744,10.0395497 L16.6254944,10.0395497 L16.6254944,13.9677517 L18.9765744,13.9677517 C21.7584423,13.9677517 24,16.2093094 24,18.9765744 C24,21.7657438 21.7584423,24 18.9765744,24 C16.2020079,24 13.9604503,21.7657438 13.9604503,18.9765744 L13.9604503,16.6327959 L10.0468512,16.6327959 L10.0468512,18.9765744 C10.0468512,21.7657438 7.79799209,24 5.03072711,24 C2.24885914,24 0,21.7657438 0,18.9765744 C0,16.2093094 2.24885914,13.9677517 5.03072711,13.9677517 L7.38180712,13.9677517 L7.38180712,10.0395497 L5.03072711,10.0395497 C2.26346212,10.0614542 0,7.79799209 0,5.01612413 C0,2.24885914 2.24885914,0 5.03072711,0 C7.79799209,0 10.0468512,2.24885914 10.0468512,5.01612413 L10.0468512,7.38180712 L13.9604503,7.38180712 L13.9604503,5.01612413 C13.9458473,2.24885914 16.2020079,0 18.9765744,0 C21.7584423,0 24,2.24885914 24,5.01612413 C24,7.79799209 21.7584423,10.0395497 18.9765744,10.0395497 Z M10.0468512,10.0468502 L10.0468512,13.9677508 L13.9677519,13.9677508 L13.9677519,10.0468502 L10.0468512,10.0468502 Z M7.38180712,5.00882263 C7.38180712,3.71645878 6.33039246,2.66504411 5.0380286,2.66504411 C3.72376027,2.66504411 2.6723456,3.71645878 2.6723456,5.00882263 C2.6723456,6.32309096 3.72376027,7.37450563 5.0380286,7.37450563 L7.38180712,7.37450563 L7.38180712,5.00882263 Z M7.38180712,18.9765744 L7.38180712,16.6327959 L5.0380286,16.6327959 C3.72376027,16.6327959 2.6723456,17.6842105 2.6723456,18.9765744 C2.6723456,20.2908427 3.72376027,21.3422574 5.0380286,21.3422574 C6.33039246,21.3422574 7.38180712,20.2908427 7.38180712,18.9765744 Z M16.6254944,18.9765744 C16.6254944,20.2908427 17.676909,21.3422574 18.9692729,21.3422574 C20.2835412,21.3422574 21.3349559,20.2908427 21.3349559,18.9765744 C21.3349559,17.6842105 20.2835412,16.6327959 18.9692729,16.6327959 L16.6254944,16.6327959 L16.6254944,18.9765744 Z M16.6254944,7.37450563 L18.9692729,7.37450563 C20.2835412,7.37450563 21.3349559,6.32309096 21.3349559,5.00882263 C21.3349559,3.71645878 20.2835412,2.66504411 18.9692729,2.66504411 C17.676909,2.66504411 16.6254944,3.71645878 16.6254944,5.00882263 L16.6254944,7.37450563 Z"></path>
+          </g>
+        </svg>
+        Mac Free Download</a>
+      <p>No affiliation with YouTube</p>
+    </footer>
```

Comment:
The part feels a little cramped, can we expand the footer session? Maybe add a bit of description? 

The footer should just include some contact info, maybe some copyright stuff.